theme/tutorial.less

```diff
@@ -221,6 +221,15 @@ body#docs.tutorial {
     width: 100%;
 }
 
+.ui.modal {
+    .ams-embed, .yt-embed {
+        width: 100%;
```

Comment:
```suggestion
```
I think this one can go too actually? or at least can set it to 80%; the 80% is less than 100% after all so having the max-width always less than this is a little confusing

Reply:
oh yeah, that makes sense. thanks!